src/main/resources/config/reflectionconfig-java.json

```diff
@@ -69,4 +87,48 @@
     "name":"java.nio.ByteOrder",
     "methods":[{"name":"nativeOrder","parameterTypes":[] }]
   }
+,
+  {
+    "name":"java.nio.file.attribute.PosixFilePermission",
```

Comment:
This is probably not available on Windows

Reply:
I can move it to linux/mac

Reply:
However, `java.nio.file.attribute.PosixFilePermission` is a valid class from `java.nio.file.attribute`, that should be available in all platforms. 
For now I'll leave it as is.